pytest.ini

```diff
@@ -10,3 +10,4 @@ filterwarnings =
     ignore:can't resolve package from __spec__ or __package__, falling back on __name__ and __path__:ImportWarning
     once:the imp module is deprecated in favour of importlib.*:DeprecationWarning
     once:the imp module is deprecated in favour of importlib.*:PendingDeprecationWarning
+    ignore:assertions
```

Comment:
Maybe limit this filter to the `pytest.PytestConfigWarning` category?

Reply:
Good idea.  I'm going to let the current tests finish, and get back to this later.

Reply:
I updated the filter with a longer part of the message and with the `pytest.PytestConfigWarning` category.